src/views/Playlist/PrivatePlaylist.vue

```diff
@@ -0,0 +1,45 @@
+<script lang="ts" setup>
+import { Configuration, GetTrackCollectionModel, TrackCollectionApi } from '@bcc-code/bmm-sdk-fetch';
+import { computed, ref } from 'vue';
+
+const props = defineProps<{
+	id: string
+}>()
+
+const trackCollection = ref<GetTrackCollectionModel>({})
+const sortedTracks = computed(() => {
```

Comment:
I don't think we need to sort them. They come in the right order from the server.

Reply:
Right order in this case means the order the user put them in.
The order property you're using is meant for within an album.